src/ui/common/vite.config.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference types="vitest" />
```

Comment:
Triple slash is something that vitest requires. It may look a bit strange but you can learn more here:
https://github.com/vitest-dev/vitest/issues/1019